### ISSUE TITLE
global url_prefix support in future

### ISSUE THREAD

**Issue by pytomtoto** [songtao-git/wings-sanic#2]
global url_prefix support in future include swagger .

**Comment by songtao-git** [comment]
Can be more detail info？

**Comment by pytomtoto** [comment]
just like Blueprint
```
domain = wings_sanic.WingsBluePrint('domain', url_prefix='/domain/v1/demo')
```
can't change the route of swagger now.

**Comment by pytomtoto** [comment]
I hava a location in nginx like this
```
location /domain { 
             proxy_pass http://domain;
}
```
so need add a global url_prefix then get /domain/swagger

**Comment by songtao-git** [comment]
Hi pytomtoo,  fixed in 0.5.22，you can add 'GLOBAL_URL_PREFIX'='/domain/v1/demo' into your settings.
